metadata.json

```diff
@@ -29,7 +29,7 @@
     {
       "operatingsystem": "CentOS",
       "operatingsystemrelease": [
-        "7",
+        "7", 
```

Comment:
could we remove this trailing whitespace?

Reply:
Thanks Gavin - I made this change in the latest commit.